core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -455,40 +469,35 @@ class TestCaseWriter {
                 when (bodyString?.first()) {
                     '[' -> {
                         // This would be run if the JSON contains an array of objects.
-                        // Only assertions on array size are supporte at the moment.
                         val resContents = Gson().fromJson(res.getBody(), ArrayList::class.java)
                         lines.add(".body(\"size()\", equalTo(${resContents.size}))")
-                        if (resContents.size > 0) {
-                            resContents.forEachIndexed { index, value ->
-                                val test_i = index
-                                val printableTh = handleFieldValues(value)
-                                if (printableTh != "null"
-                                        && printableTh != "NotCoveredYet"
-                                        && !printableTh.contains("logged")
-                                ) {
-                                    lines.add(".body(\"get($test_i)\", $printableTh)")
+                        //resContents.sortBy { it.toString() }
+                        //assertions on contents
+                        if(resContents.size > 0){
+                            if(resContents.first()::class == LinkedTreeMap::class) resContents.sortBy { it.toString() }
```

Comment:
this would break as soon as Gson will use something different than a LinkedTreeMap, isn't it? i think it would be better to rather check if the class is implementing "Map"

Reply:
Change to Map.

I was mostly concerned about using methods specific to LinkedTreeMap, but since I'm not using any, Map should work the same.